bpdm-pool-api/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/api/model/request/LegalEntityPropertiesSearchRequest.kt

```diff
@@ -26,9 +26,25 @@ import io.swagger.v3.oas.annotations.media.Schema
 @Schema(description = "Contains keywords used for searching in legal entity properties")
 data class LegalEntityPropertiesSearchRequest constructor(
     @field:Parameter(description = "Filter legal entities by name")
-    val legalName: String?
+    val legalName: String?,
+
+    @field:Parameter(description = "Filter business partners by CX-BPN.")
+    val id: String?,
+
+    @field:Parameter(description = "Filter business partners by street name.")
+    val street: String?,
```

Comment:
Street is not a single field. We should talk about what is the expectation here if somebody puts something in that field. Should it search for partial matches in all street DB fields?